initfiles/componentfiles/configxml/validateAll.xsl

```diff
@@ -42,7 +42,7 @@ xmlns:seisint="http://seisint.com" exclude-result-prefixes="seisint">
       </xsl:if> 
     </xsl:for-each>
     <xsl:variable select="@name" name="elem1"/>
-        <xsl:if test="(translate($elem1,'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_',''))" >
+        <xsl:if test="(translate($elem1,'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-',''))" >
```

Comment:
are dropzone names the only artifacts affected by this change? If not, is it ok to allow '-' in other @name attributes?
Also, will this logic allow leading dashes in the name?

Reply:
Right, and I'm pointing out a potential side affect. Please make sure the side affects are reasonable. @kenrowland